### DISCUSSION TITLE
the rigth cables / wires on ali

### DISCUSSION THREAD

**Discussion by maromme** [raomin/ESPAltherma#278] [General]
Hi,
I've got an aliexpress seller for the proper wires (he is doing individual wire configurations on request - just write a message and he is adding the wire to the online store)

JST-EH to dupont:
https://www.aliexpress.com/item/1005005474524200.html

JST-PA to dupont
https://www.aliexpress.com/item/1005005465411765.html

I've got a couple in spare. Just ask...

**Comment by goldendel** [comment]
Hi @maromme 

Thanks for posting this.  It's really helpful.  I'm not really familiar with JST connectors (I've only just learned it stands for Japanese Solderless Terminal).  However, I can see from the two links you have listed, the JST-PA to dupont has female connectors.  Does that mean the PA 2.0 connector connects to the HY 2.0/GROVE port next to USB C rather than the 8 port terminal?

![image](https://github.com/raomin/ESPAltherma/assets/12022664/608132ca-7c6d-4eaa-b09b-272ab5b9c7f6)

Cheers,

Tim

**Comment by remycrochon** [comment]
Bonjour et merci pour ces infos.

**Comment by maromme** [comment]
In my setup I needed the female connectors. Nothing to do without the groove port.

**Comment by goldendel** [comment]
Aha. Thanks for straightening me out @maromme.  It makes more sense to me that it's connecting in on the 8 port connector.

I'm having problems with WiFi strength once in the Altherma casing.  It's been suggested I put the M5Stick CPlus 1.1 I have outside the casing.  I've noticed there is a conduit cable from the Altherma into my garage, closer to the Wireless Access Point (I too have a mesh).  So, I'm thinking about running a cable in the conduit to connect directly to the X10A pins.

Do you (or anyone else know if there are transmission limits on the dupont cabling?  I think I'm going to require 3 or 5 metres of cable.

Thanks in advance for your advice.

Cheers,

Tim

**Reply by maromme** [reply]
I just would give it a try. If you nee any of those wires I've got some spare...

**Reply by goldendel** [reply]
Thanks @maromme.  I haven't measured it precisely yet, but I think it will be between 3 and 5 metres.  Do you have 4 cables that long?

Cheers,

Tim

**Reply by maromme** [reply]
No mine are shorter ones. Like in the Ali links above

**Comment by mihu-ov** [comment]
JST EH connectors are used for LiPo battery balancer connectors so you may be able to find them at R/C model shops with better lead times than from Aliexpress. Ask for 4s (4 battery cells in series) JST-EH balancer cables or extension cords. Just as an example, I got mine delivered next day from https://www.d-m-t.at/elektrozubehoer/kabel-steckverbindungen/silikonkabel-litzen-adapterkabel/adapterkabel-v-kabel-fuer-akkus/balancer-verlaengerungskabel-4s-jst-eh-30-cm-detail

**Comment by goldendel** [comment]
> No mine are shorter ones. Like in the Ali links above

Thanks for the offer and confirming.  I'll measure up properly and order some longer ones M-F Duponts.

**Reply by mihu-ov** [reply]
Serial communication between heat pump and ESP32 is with 3.3V/5V level signals in an electrical noisy environment. You may get away with 3-5m cable length but I would not bet on it.

**Reply by goldendel** [reply]
Thanks. I'm not betting on it, but I do need to try it. Putting the ESP32 into the Faraday cage that is my monobloc is effectively blocking all wireless service giving me no feasible data as soon as I put on the case.
